File: server/definition/application.go
package definition

import "gorm.io/gorm"

type Application struct {
	gorm.Model

	Id                       string `json:"id"`
	Username                 string `json:"username"` // Ingame username
	Age                      int    `json:"age"`
	WhereDidYouFindTheServer string `json:"where_did_you_find_the_server"`
	Bio                      string `json:"bio"`
	Status                   int    `json:"status"` // 0: Pending, 1: Rejected, 2: Approved
}

File: mod/src/main/kotlin/site/remlit/apply/model/Application.kt
package site.remlit.apply.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Application(
	val id: String,
	val username: String,
	val age: Int,
	@SerialName("where_did_you_find_the_server")
	val whereDidYouFindTheServer: String,
	val bio: String,
	val status: Int
)